interface_android.go

```diff
@@ -422,3 +417,9 @@ func nameToFlags(name string) (net.Flags, error) {
 
 	return linkFlags(*(*uint32)(unsafe.Pointer(&ifr[syscall.IFNAMSIZ]))), nil
 }
+
+// Returns the API level of the device we're actually running on, or -1 on failure.
+// The returned value is equivalent to the Java Build.VERSION.SDK_INT API.
+func androidApiLevel() int {
+	return int(C.android_get_device_api_level())
```

Comment:
It seems that here we can use once.Do to perform a call once, and then save the result in a global variable to avoid the overhead of each CGO call.